astropy/nddata/utils.py

```diff
@@ -11,6 +11,11 @@
            'overlap_slices']
 
 
+class NoOverlapError(ValueError):
```

Comment:
I only now realize that "pos" was probably meant to represent and index and not a coordinate. I've always thought of it as a coordinate, and I may have been wrong. That's why I chose "Value" and not "Index" for the error.